python-checks/src/test/resources/checks/nosonarComment.py

```diff
@@ -0,0 +1,27 @@
+def divide(numerator, denominator):
+# Noncompliant@+1
```

Comment:
```suggestion
# Noncompliant@+1 {{Is #NOSONAR used to exclude false-positive or to hide real quality flaw?}}
```
Besides checking the location, you can also check for the message of an issue.

Reply:
Added this message to the test file as well as another Noncompliant@+1 above this line since, due to the current way to detect NOSONAR marker, it will also raise a issue